homeassistant/components/unifiprotect/data.py

```diff
@@ -88,6 +96,51 @@ def max_events(self) -> int:
         """Max number of events to load at once."""
         return self._entry.options.get(CONF_MAX_MEDIA, DEFAULT_MAX_MEDIA)
 
+    @property
+    def ignored_macs(self) -> set[str]:
+        """Set of ignored MAC addresses."""
+
+        if self._ignored_macs is None:
+            self._ignored_macs = convert_mac_list(
+                self._entry.options.get(CONF_IGNORED, "")
+            )
+
+        return self._ignored_macs
+
+    @callback
+    def async_get_changed_options(self, entry: ConfigEntry) -> dict[str, Any]:
+        """Get changed options for when entry is updated."""
+
```

Comment:
You might be able to simplify this a bit and offload the heavy lifting to native code with something like
```
changed = dict(set(new.items()) - set(old.items()))
```